jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricher.java

```diff
@@ -100,32 +120,32 @@ public void visit(ContainerBuilder containerBuilder) {
         super.create(platformMode, builder);
     }
 
-    private Probe discoverWildflyJARHealthCheck(boolean isReadiness) {
-
+    private Probe discoverWildflyJARHealthCheck(String path, int initialDelay, int failureThreshold, int successThreshold, int periodSeconds) {
         if (isAvailable()) {
-            Integer port = getPort();
+            int port = asInt(getConfig(Config.PORT));
             if (port <= 0) {
                 return null;
             }
-            int initialDelay = isReadiness ? getReadinessInitialDelay() : getLivenessInitialDelay();
-            // scheme must be in upper case in k8s
-            String scheme = getScheme().toUpperCase();
-            String path = isReadiness ? getReadinessPath() : getLivenessPath();
-
             return new ProbeBuilder()
-                    .withNewHttpGet().withNewPort(port).withPath(path).withScheme(scheme).endHttpGet()
-                    .withFailureThreshold(getFailureThreshold())
-                    .withSuccessThreshold(getSuccessThreshold())
-                    .withInitialDelaySeconds(initialDelay).build();
+                    .withNewHttpGet().withNewPort(asInteger(getConfig(Config.PORT)))
+                    .withPath(path)
+                    .withScheme(getConfig(Config.SCHEME).toUpperCase())
+                    .endHttpGet()
+                    .withFailureThreshold(failureThreshold)
+                    .withSuccessThreshold(successThreshold)
+                    .withInitialDelaySeconds(initialDelay)
+                    .withPeriodSeconds(periodSeconds)
+                    .build();
         }
         return null;
     }
 
     private boolean isAvailable() {
-        if (isProbeEnforced()) {
+        boolean isProbeEnforced = asBoolean(getConfig(Config.ENFORCE_PROBES));
```

Comment:
is this variable used elsewhere? why not just evaluate in the if clause?

Reply:
This wasn't fixed
We still have a local variable declaration that is not needed. Why aren't we just consuming its instantiation value?

```java
        if (asBoolean(getConfig(Config.ENFORCE_PROBES))) {
            return true;
        }

```